web/massastation/src/pages/Store/StoreSection/StorePlugin.tsx

```diff
@@ -9,6 +9,13 @@ interface StorePluginProps {
   plugin: IMassaStore;
   refetch: () => void;
 }
+
+const LoadingDownload = () => (
+  <div className={`rounded-full animate-pulse blur-sm`}>
+    <FiDownload />
+  </div>
+);
```

Comment:
I wont complain if you decide to keep as it is here but we, as a team, we said that we would use regular functions for that.

```suggestion
function LoadingDownload() {
  return ( 
    <div className={`rounded-full animate-pulse blur-sm`}>
      <FiDownload />
    </div>
  );
}
```

Reply:
sorry about that, I just changed it